geode-core/src/main/java/org/apache/geode/cache/RegionShortcut.java

```diff
@@ -192,30 +192,36 @@
    * {@link DataPolicy} to {@link DataPolicy#PERSISTENT_REPLICATE} and {@link EvictionAttributes}
    * are set to {@link EvictionAlgorithm#LRU_HEAP} with {@link EvictionAction#OVERFLOW_TO_DISK}.
    */
-  LOCAL_PERSISTENT_OVERFLOW,
+  LOCAL_PERSISTENT_OVERFLOW("PERSISTENT_REPLICATE"),
 
   /**
    * A PARTITION_PROXY has no local state and forwards all operations to a PARTITION or a
    * PARTITION_PERSISTENT that exists in its peers. The actual RegionAttributes for a
    * PARTITION_PROXY region set the {@link DataPolicy} to {@link DataPolicy#PARTITION} and the
    * local-max-memory to 0.
    */
-  PARTITION_PROXY,
+  PARTITION_PROXY("PARTITION"),
   /**
    * A PARTITION_PROXY_REDUNDANT has no local state and forwards all operations to a
    * PARTITION_REDUNDANT or a PARTITION_REDUNDANT_PERSISTENT that exists in its peers. The actual
    * RegionAttributes for a PARTITION_PROXY_REDUNDANT region set the {@link DataPolicy} to
    * {@link DataPolicy#PARTITION}, the local-max-memory to 0, and the redundant-copies to 1.
    */
-  PARTITION_PROXY_REDUNDANT,
+  PARTITION_PROXY_REDUNDANT("PARTITION"),
   /**
    * A REPLICATE_PROXY has no local state and forwards all operations (except queries) to a
    * REPLICATE or a REPLICATE_PERSISTENT that exists in its peers. Queries will be executed on this
    * PROXY region. The actual RegionAttributes for a REPLICATE_PROXY region set the
    * {@link DataPolicy} to {@link DataPolicy#EMPTY} and {@link Scope} to
    * {@link Scope#DISTRIBUTED_ACK}.
    */
-  REPLICATE_PROXY;
+  REPLICATE_PROXY("EMPTY");
+
+  private String dataPolicy;
```

Comment:
Why not have this be the actual DataPolicy object, instead of the string name of the object?